CODIGO/BabelUI.bas

```diff
@@ -956,45 +956,35 @@ On Error GoTo HandleUpdateIntSetting_Err
             Call SaveSetting("OPCIONES", "UseExperimentalUI", Value)
         Case eEnableMusic
             If Value = 0 Then
-                Sound.Music_Stop
-                Musica = CONST_DESHABILITADA
+                ao20audio.stopallplayback
+                ao20audio.MusicEnabled = 0
             Else
-                Musica = CONST_MP3
-                Sound.NextMusic = MapDat.music_numberHi
-                Sound.Fading = 100
+                ao20audio.MusicEnabled = 0
```

Comment:
both cases are setting MusicEnabled to 0

Reply:
Nice catch. Resolved in next patch.